CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 * Add your own contribution below
 
+## 4.3.3
+
+* Add actual file URL to `danger.import_dangerfile(github:)` (supports private repositories and GitHub Enterprise now)
```

Comment:
From an end-users's perspective this doesn't make too much sense, maybe just go with? 

> `danger.import_dangerfile(github:)`now supports private repositories and GitHub Enterprise - hotbott